src/charactersheet/services/character/status_service_components/inspiration_component.js

```diff
@@ -16,24 +17,40 @@ export function InspirationStatusServiceComponent() {
 
     self.statusIdentifier = 'Status.Inspired';
 
-    self.init = function() {
-        Notifications.otherStats.inspiration.changed.add(self.dataHasChanged);
+    self.init = async function() {
 
-        // Calculate the first one.
-        self.dataHasChanged();
+        await self.load();
+        self.setUpSubscriptions();
     };
 
+    self.setUpSubscriptions = () => {
+        Notifications.otherstats.changed.add(self.dataHasChanged);
+        Notifications.coreManager.changing.add(self.clear);
+    };
+
+    self.clear = () => {
+        self._removeStatus();
+    };
+
+    self.load = async () => {
+        if (ko.utils.unwrapObservable(CoreManager.activeCore().type.name) !== 'character') {
+            return;
+        }
+        var key = CoreManager.activeCore().uuid();
+        var otherStatsResponse = await OtherStats.ps.read({uuid: key});
```

Comment:
Can't this just use the `model.load` thing?

Reply:
We can. The entire otherStats isn't stored in the inspiration component, and we don't need the full features of OtherStats, so we grab the data we need and discard the rest. 

However, this is a bug that need to be addressed, since it doesn't update when character changes.